components/suggestion/switch_channel_provider.jsx

```diff
@@ -126,6 +128,21 @@ class SwitchChannelSuggestion extends Suggestion {
             );
         }
 
+        let tag = null;
+        if (channel.type === Constants.DM_CHANNEL) {
+            var teammate = Utils.getDirectTeammate(channel.id);
```

Comment:
We probably shouldn't be adding another usage of `Utils.getDirectTeammate` since that accesses the store directly, but that's 100% nitpicking, so it's too late to change now.

Reply:
Hmm.. Ya I am going to use old component compatibility to get away with this. It would take a bunch of work to get the info I need down to this component.